### PR TITLE
[Docs] Update "Customizing Forms"

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no/
| License         | MIT

### CODE REVIEW

File: docs/customization/form.rst [hunk 1]
```diff
@@ -202,7 +206,6 @@ you will also have to set up an event listener and then remove the field:
             ;
 
             ...
-
```
Comment (CoderMaggie) [inline docs/customization/form.rst]: Why?

File: docs/customization/form.rst [hunk 2]
```diff
@@ -105,8 +104,8 @@ As a result you will get the ``Sylius\Bundle\CustomerBundle\Form\Type\CustomerPr
     Of course remember that you need to render the new fields you have created,
     and remove the rendering of the fields that you have removed **in your views**.
 
-In our case you will need to copy the original template from `vendor/sylius/sylius/src/Sylius/Bundle/ShopBundle/Resources/views/Account/profileUpdate.html.twig`
-to `templates/bundles/SyliusShopBundle/Account/` and add the fields inside the copy
+In our case you will need to copy the original template from ``vendor/sylius/sylius/src/Sylius/Bundle/ShopBundle/Resources/views/Account/profileUpdate.html.twig``
+to ``templates/bundles/SyliusShopBundle/Account/`` and add the fields inside the copy
```
Comment (CoderMaggie) [inline docs/customization/form.rst]:
```suggestion
to ``templates/bundles/SyliusShopBundle/Account/`` and add the fields inside the copy.
```

File: docs/customization/form.rst [hunk 3]
```diff
@@ -19,6 +19,11 @@ You can:
 How to customize a Form?
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. tip::
+
+    You can browse the full implementation of this example on GitHub
+    on the `<https://github.com/Sylius/Customizations/pull/8>`_
```
Comment (CoderMaggie) [inline docs/customization/form.rst]: How does this look like when you render it?

File: docs/customization/form.rst (hunk 1 above)
Comment (Tomanhez) [reply docs/customization/form.rst]: I think that now looks like better :D ?

File: docs/customization/form.rst (hunk 3 above)
Comment (Tomanhez) [reply docs/customization/form.rst]: I added picture in my PR. It is not enough?

File: docs/customization/form.rst [hunk 4]
```diff
@@ -55,35 +60,29 @@ As a result you will get the ``Sylius\Bundle\CustomerBundle\Form\Type\CustomerPr
     use Symfony\Component\Form\AbstractTypeExtension;
     use Symfony\Component\Form\Extension\Core\Type\TextType;
     use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\Validator\Constraints\Length;
 
     final class CustomerProfileTypeExtension extends AbstractTypeExtension
```
Comment (lchrusciel) [inline docs/customization/form.rst]: However `declare(strict_types=1)` could be added

**Comment by GSadee** [comment]
Thank you, Tomasz! :tada:

**Pull request merged**